### PR TITLE
Product type Unit test

### LINKED ISSUE #1131: Add Product Type Unit Tests
Please use the following example at  as a template:
https://github.com/DefectDojo/django-DefectDojo/blob/dev/tests/Product_unit_test_example.py

Implement the following tests:
- Add
- Edit
 
Master List and Overview: https://docs.google.com/document/d/1dSS7tWuQukAX3VWtzPAlOfFNoqLQh-mBHBfmsu2WlU0/edit?usp=sharing

### PR DESCRIPTION
**Pull request by dr3dd589**
fix #1131 
Implemented test for the following:-
   - Add product type
   - Edit product type
   - Delete product type

- [x] Your code is flake8 compliant (DefectDojo's code isn't currently flake8 compliant, but we're trying to correct that.)
- [ ] If this is a new feature and not a bug fix, you've included the proper documentation in the ReadTheDocs documentation folder. https://github.com/DefectDojo/Documentation/tree/master/docs or provide feature documentation in the PR.
- [ ] Model changes should include the necessary migrations in the dojo/dd_migrations folder.
- [ ] Add applicable tests to the unit tests.

### CODE REVIEW

File: tests/Product_type_unit_test.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
+from selenium import webdriver
+import unittest
+import re
+import sys
+
+
+class ProductTest(unittest.TestCase):
+    def setUp(self):
+        # change path of chromedriver according to which directory you have chromedriver.
+        self.driver = webdriver.Chrome('/home/dr3dd/gsoc/chromedriver')
```
Comment (Maffooch) [inline tests/Product_type_unit_test.py]: For purposes of testing with Travis, the chromedriver needs to be in the same directory as the file being tested. See the new unit test example for reference.

File: tests/Product_type_unit_test.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        self.verificationErrors = []
+        self.accept_next_alert = True
+
+    def login_page(self):
+        driver = self.driver
+        driver.get(self.base_url + "login")
+        driver.find_element_by_id("id_username").clear()
+        driver.find_element_by_id("id_username").send_keys('admin')
+        driver.find_element_by_id("id_password").clear()
+        driver.find_element_by_id("id_password").send_keys('admin')
```
Comment (Maffooch) [inline tests/Product_type_unit_test.py]: Credentials have changed to play nicely with Travis.  See the new unit test example for reference.

File: tests/Product_type_unit_test.py [hunk 3]
```diff
@@ -0,0 +1,72 @@
+from selenium import webdriver
+import unittest
+import re
+import sys
+
```
Comment (Maffooch) [inline tests/Product_type_unit_test.py]: With the modification to login credentials, "import os" needs to be added here.

**Comment by dr3dd589** [comment]
@Maffooch done. thanks!

**Comment by devGregA** [comment]
@Maffooch please confirm when it's good to merge.

**Comment by Maffooch** [comment]
Confirmed to merge.

**Comment by devGregA** [comment]
Great stuff @dr3dd589 which one would you like next?

**Comment by dr3dd589** [comment]
start working on this #1133.

**Pull request merged**